### ISSUE TITLE
Stop using 'should' as a verb for joining mandatory conditions

### ISSUE THREAD

**Issue by turt2live** [matrix-org/matrix-doc#2799]
We describe SHOULD as strongly recommended, and occasionally forget to use the right case in doing this. When we use the wrong case we tend to mean one of two things: the right case, or that previously established context is being furthered clarified. 

An example is the fallback representation of replies: clients MUST include a fallback when they support rich replies, and they are required to use the plain text fallback format. The sentence which follows, 'the formatted_body should use the following template' is not actually making the template a strong recommendation, it's saying that given the plain text format previously described, the result of the HTML format for the same event would be the following. Should is the right word in this case, but extremely hard to understand. We shouldn't have the spec be cryptic.

**Comment by ShadowJonathan** [comment]
Maybe [RFC 2119](https://tools.ietf.org/html/rfc2119) could be used here to disambiguate between those terms?

**Comment by deepbluev7** [comment]
Agreed. It took me a long while to understand, why the fallback is required to look that way, since I didn't understand the difference between SHOULD and should. I somehow doubt Travis scales well enough to spend 30 minutes to explain that to everyone, who has the same issue.
